src/libutil/args.cc

```diff
@@ -249,6 +247,11 @@ nlohmann::json Args::toJSON()
             j["arity"] = flag->handler.arity;
         if (!flag->labels.empty())
             j["labels"] = flag->labels;
+        // FIXME With C++23 use std::optional::tranform
```

Comment:
```suggestion
        // TODO With C++23 use std::optional::tranform
```

Not an actual problem, just something we may do.